pages/DSNP/Announcements.md

```diff
@@ -4,18 +4,18 @@ name: Announcements
 route: /DSNP/Announcements
 ---
 
-
-# TODO
-
-I took out signatures from this level. Do we still need them for DSNP 1.0?
-
-
 # Announcements Overview
 
 Announcements are content or reference to content that communicate new user activity to the rest of the network.
 Announcements are associated with an [Identifier](/DSNP/Identifiers) which can be validated as creator of the announcement.
 Depending on the implementation Announcements may be published directly to the network, included in [Batch Publication Files](/DSNP/BatchPublications), or some combination of those two.
 
+## Announcement Validation
+
+There is no guarantee that, at time of creation, a given announcement will be from the `fromId` claimed in the announcement.
+The reader MUST perform a validation of the announcement at read time to ensure authenticity.
+Implementations MUST provide a way to validate what [signatures](/DSNP/Signatures) are associated with a given [identifier](/DSNP/Identifiers).
+
```

Comment:
@wilwade  how does that, moved around to read better

Reply:
Nice! Let's get this PR merged and we can make a new one for additional changes.